<ui version="4.0" >
 <class>DocumentationPanel</class>
 <widget class="QWidget" name="DocumentationPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>116</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Documentation</string>
  </property>
  <layout class="QVBoxLayout" name="ui_verticalLayout" >
   <property name="spacing" >
    <number>3</number>
   </property>
   <property name="margin" >
    <number>1</number>
   </property>
   <item>
    <widget class="QTextBrowser" name="ui_textBrowser" />
   </item>
  </layout>
  <action name="ui_backwardAction" >
   <property name="icon" >
    <iconset resource="../resources/gui.qrc" >
     <normaloff>:/backwardIcon</normaloff>:/backwardIcon</iconset>
   </property>
   <property name="text" >
    <string>Backward</string>
   </property>
  </action>
  <action name="ui_forwardAction" >
   <property name="icon" >
    <iconset resource="../resources/gui.qrc" >
     <normaloff>:/forwardIcon</normaloff>:/forwardIcon</iconset>
   </property>
   <property name="text" >
    <string>Forward</string>
   </property>
  </action>
  <action name="ui_homeAction" >
   <property name="icon" >
    <iconset resource="../resources/gui.qrc" >
     <normaloff>:/homeIcon</normaloff>:/homeIcon</iconset>
   </property>
   <property name="text" >
    <string>Home</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/gui.qrc" />
 </resources>
 <connections/>
</ui>
